.github/workflows/core-plugins-build-and-release.yml

```diff
@@ -7,6 +7,7 @@ on:
         type: choice
         options:
           - grafana-testdata-datasource
+          - tempo
```

Comment:
we should add here azure monitor and parca too and do an initial release cc/ @aangelisc